### PR TITLE
kernel-install: add page

### PR DESCRIPTION
**Pull request by sgrverma23**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by sgrverma23** [comment]
@owenvoke @bl-ue Please review this PR to add `kernel-install `command page.

**Comment by sgrverma23** [comment]
Please add hacktoberfest label as well.

**Comment by sgrverma23** [comment]
@gutjuri @reinhart1010 @kbdharun Please review the PR and add the label of hacktoberfest as well.

**Comment by kbdharun** [comment]
> @gutjuri @reinhart1010 @kbdharun Please review the PR and add the label of hacktoberfest as well.

Welcome to tldr and thanks for your contribution. I will review this PR soon.

Btw don't ping maintainers asking for review on new PRs, we will always check out open PRs once every couple of days. This PR is eligible without a hacktoberfest label tagged to it separately as we have added Hacktoberfest topics to our repository topic labels. Once it gets merged after being approved by 2 maintainers, the PR will be considered as a contribution after a 7 day waiting period.

File: pages/linux/kernel-install.md
Comment (gutjuri) [inline pages/linux/kernel-install.md:8]:
```suggestion
`sudo kernel-install add {{kernel-version}} {{kernel-image}} {{initrd-file ...}}`
```

File: pages/linux/kernel-install.md
Comment (kbdharun) [inline pages/linux/kernel-install.md:4]:
```suggestion
> More information: <https://manned.org/kernel-install.8>.
```
We use Manned as the default fallback when there isn't an author's documentation. (And this command isn't an Arch Linux project so we can use manned)

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Please sign the CLA for the PR to go through.

File: pages/linux/kernel-install.md
Comment (gutjuri) [inline pages/linux/kernel-install.md:14]:
```suggestion
- Show various paths and parameters that have been configured or auto-detected:
```

**Review by gutjuri — CHANGES_REQUESTED** [review]

File: pages/linux/kernel-install.md
Comment (kbdharun) [reply pages/linux/kernel-install.md:8]: Seem like we both reviewed at same time 😅 , will resolve this suggestion alone. The other one LGTM.

File: pages/linux/kernel-install.md
Comment (sgrverma23) [reply pages/linux/kernel-install.md]: Change added.

File: pages/linux/kernel-install.md
Comment (sgrverma23) [reply pages/linux/kernel-install.md]: Change added.

File: pages/linux/kernel-install.md
Comment (sgrverma23) [reply pages/linux/kernel-install.md]: Change added.

File: pages/linux/kernel-install.md
Comment (sgrverma23) [reply pages/linux/kernel-install.md]: Change added.

**Comment by sgrverma23** [comment]
I have signed the CLA but it is not reflecting. @kbdharun 

image

**Review by gutjuri — APPROVED** [review]

**Comment by kbdharun** [comment]
> I have signed the CLA but it is not reflecting. @kbdharun

image

I can confirm you have signed the CLA (checked it in the admin interface). But the commits you made might not have the same email address as your GitHub account/your email isn't configured locally. Feel free to try out these steps https://github.com/tldr-pages/tldr/blob/main/contributing-guides/git-terminal.md#changing-the-email-of-any-commits after setting the name and email globally using:

```
git config --global user.name "Your name"
git config --global user.email "your@email.com"
```
Also, add the mail to your current GitHub account (if not already done).

**Comment by sgrverma23** [comment]
@kbdharun I have tried to signed the agreement again. 
Can you also approve the PR please.

**Comment by kbdharun** [comment]
Seems like the CLA issue still seems to persist, can you delete the signed CLA and resign it?

**Review by kbdharun — APPROVED** [review]

**Comment by sgrverma23** [comment]
I have deleted the signed CLA and signed it again but still the same issue :(

**Comment by sgrverma23** [comment]
There are some issues with this PR and CLA so I have opened a new PR and added the changes.  Please review it. @kbdharun @gutjuri 
https://github.com/tldr-pages/tldr/pull/10923

**Pull request closed without merging**
